modules/installation/pages/proxy-setup.adoc

```diff
@@ -220,18 +220,56 @@ If you do not use the reactivation key, you will need to re-register all the cli
 
 
 
-.Procedure: Replacing a Proxy and Keeping the Clients Registered
+.Procedure: Replacing a Traditional Proxy and Keeping the Clients Registered
 . Before starting the migration, save the data from the old proxy, if needed.
   Consider copying important or custom data to a central place that can also be accessed by the new proxy.
-. Shut down the proxy.
+. Shut down the old proxy.
 . Install a new {productname} Proxy.
   For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
 . In the {productname} {webui}, select the newly installed {productname} Proxy, and delete it from the systems list.
-. In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
+. In the {webui}, create a reactivation key for the old proxy system.
+  On the [guimenu]``System Details`` tab of the old proxy click [guimenu]``Reactivation``.
   Click [guimenu]``Generate New Key``, and make a note of the new key.
 . Register the proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
-  In the bootstrap script set the reactivation key with the [systemitem]``ACTIVATION_KEYS`` parameter.
-  For more information about using reactivation keys, see xref:client-configuration:activation-keys.adoc[].
+  In the bootstrap script, set the reactivation key with the [systemitem]``ACTIVATION_KEYS`` parameter.
```

Comment:
For the sake of consistency, REACTIVATION_KEY can be used with Traditional proxy as well. So I suggest to

```suggestion
  In the bootstrap script, set the reactivation key with the [systemitem]``REACTIVATION_KEY`` parameter.
```